Src/BootCamp.Chapter/PcFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using BootCamp.Chapter.Computer;
+
+namespace BootCamp.Chapter
+{
+    public abstract class PcFactory
+    {
+        protected Body body;
+        protected Cpu cpu;
+        protected Gpu gpu;
+        protected HardDisk disk;
```

Comment:
move fields to local scope

Reply:
I've copletely deleted them hope it is right :D